src/App.js

```diff
@@ -0,0 +1,37 @@
+let todoList = []
+
+todoList = [
+  {
+    id: 1,
+    title: 'Complete assignment 1'
+  },
+  {
+    id: 2,
+    title: 'Complete assignment 2'
+  },
+  {
+    id: 3,
+    title: 'Complete assignment 3'
+  }
+]
+
+function App() {
+  return (
+    <div>
+      <header style={{ textAlign: 'center' }}>
+        <h1>Todo List</h1>
+      </header>
+      <hr />
```

Comment:
Nice, leaving the `<header>`. It's more semantically correct to but the h1 inside the`<header>`.